### PR TITLE
Automate load test execution

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we add a build script and an additional script to
simplify running load tests with k6. The load test results will be
written to a shared metrics store where they can be inspected. We also
include an automatic warmup phase that will be tagged separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,22 @@ Space time benchmarking project with k6 tool
...
+
+ - Install dependencies and transpile typescript code into JS and bundle the project: `make build`
  - Run your test:
     - using kibana.json in assets folder: `k6 run -e USE_KIBANA_JSON=1 dist/login_test.js`
     - by passing values directly: `k6 run -e KIBANA_USERNAME=elastic -e KIBANA_PASSWORD=changeme -e KIBANA_URL="http://localhost:5620" -e KIBANA_VERSION=8.3.0 dist/login_test.js`
+## Running tests directly with k6
+
+ - Ensure that Vault is properly setup
+ - Run your test:
+    - Running all load tests: `./k6.sh`
```
Comment (dmlemeshko) [inline README.md:25]: I'm getting error:
```
[automate][~/github/kibana-k6-benchmarking]$ ./k6.sh
yarn install v1.22.18
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.09s.
yarn run v1.22.18
$ /Users/dmle/github/kibana-k6-benchmarking/node_modules/.bin/webpack
asset lens_test.js 12.8 KiB [emitted] (name: lens_test) 1 related asset
asset login_test.js 5.75 KiB [emitted] (name: login_test) 1 related asset
asset kibana.json 121 bytes [emitted] [from: assets/kibana.json] [copied]
runtime modules 1.83 KiB 8 modules
orphan modules 2.16 KiB [orphan] 6 modules
built modules 11.9 KiB [built]
  ./src/lens_test.ts + 6 modules 9.39 KiB [not cacheable] [built] [code generated]
  ./src/login_test.ts + 5 modules 2.56 KiB [not cacheable] [built] [code generated]
webpack 5.35.1 compiled successfully in 261 ms
✨  Done in 0.55s.
Cloning into '.k6'...
remote: Enumerating objects: 21, done.
remote: Counting objects: 100% (21/21), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 21 (delta 5), reused 19 (delta 3), pack-reused 0
Receiving objects: 100% (21/21), 13.24 KiB | 6.62 MiB/s, done.
Resolving deltas: 100% (5/5), done.
make[1]: go: No such file or directory
make[1]: *** [format] Error 1
make: *** [install-k6-es] Error 2
[automate][~/github/kibana-k6-benchmarking]$
```

Vault is configured. Did I miss something?

Comment (dmlemeshko) [reply README.md:25]: Some progress after installing `go` + running `go install go.k6.io/xk6/cmd/xk6@latest
`, but build is failing now:
```
[automate][~/github/kibana-k6-benchmarking]$ ./k6.sh
yarn install v1.22.18
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.09s.
yarn run v1.22.18
$ /Users/dmle/github/kibana-k6-benchmarking/node_modules/.bin/webpack
asset lens_test.js 12.8 KiB [emitted] (name: lens_test) 1 related asset
asset login_test.js 5.75 KiB [emitted] (name: login_test) 1 related asset
asset kibana.json 121 bytes [emitted] [from: assets/kibana.json] [copied]
runtime modules 1.83 KiB 8 modules
orphan modules 2.16 KiB [orphan] 6 modules
built modules 11.9 KiB [built]
  ./src/lens_test.ts + 6 modules 9.39 KiB [not cacheable] [built] [code generated]
  ./src/login_test.ts + 5 modules 2.56 KiB [not cacheable] [built] [code generated]
webpack 5.35.1 compiled successfully in 259 ms
✨  Done in 0.70s.
Cloning into '.k6'...
remote: Enumerating objects: 21, done.
remote: Counting objects: 100% (21/21), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 21 (delta 5), reused 19 (delta 3), pack-reused 0
Receiving objects: 100% (21/21), 13.24 KiB | 6.62 MiB/s, done.
Resolving deltas: 100% (5/5), done.
?   	github.com/elastic/xk6-output-elasticsearch	[no test files]
?   	github.com/elastic/xk6-output-elasticsearch/pkg/esoutput	[no test files]
make[1]: xk6: No such file or directory
make[1]: *** [build] Error 1
make: *** [install-k6-es] Error 2
```
Probably some more configuration around Go is required?

Comment (danielmitterdorfer) [reply README.md:25]: `xk6` is installed in `go env GOPATH` (typically `$HOME/go`) and expects its `bin` directory on the path. So you should add:

```
export PATH=$PATH:$(go env GOPATH)/bin
```

to your `.profile` or `.zprofile` (depending on your shell). See https://go.dev/doc/gopath_code#GOPATH for more background.

File: README.md [hunk 2]
```diff
@@ -6,8 +6,22 @@ Space time benchmarking project with k6 tool
...
+
+### Directly with k6
+
+ - Install dependencies and transpile typescript code into JS and bundle the project: `make build`
  - Run your test:
     - using kibana.json in assets folder: `k6 run -e USE_KIBANA_JSON=1 dist/login_test.js`
     - by passing values directly: `k6 run -e KIBANA_USERNAME=elastic -e KIBANA_PASSWORD=changeme -e KIBANA_URL="http://localhost:5620" -e KIBANA_VERSION=8.3.0 dist/login_test.js`
+## Running tests directly with k6
+
+ - Ensure that Vault is properly setup
```
Comment (dmlemeshko) [inline README.md:22]: I would add a line about GO installation and `export PATH=$PATH:$(go env GOPATH)/bin`

**Review by dmlemeshko — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (danielmitterdorfer) [reply README.md:22]: Added in https://github.com/elastic/kibana-k6-benchmarking/pull/12/commits/e42e62c55b8034e44e2695a78e0c8dc551819d09.

**Pull request merged**